man/systemd.special.xml

```diff
@@ -1167,6 +1167,53 @@
         </varlistentry>
       </variablelist>
     </refsect2>
+
+    <refsect2>
+      <title>Special User Slice Units</title>
+
+      <para>There are four <literal>.slice</literal> units which form the basis of the user hierarchy for
+      assignment of resources for user applications and services. See
+      <citerefentry><refentrytitle>systemd.slice</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+      for details about slice units and the documentation about
+      <ulink url="https://systemd.io/DESKTOP_ENVIRONMENTS">Desktop Environments</ulink>
+      for further information.</para>
+
+      <variablelist>
+        <varlistentry>
+          <term><filename>-.slice</filename></term>
+          <listitem>
+            <para>The root slice is the root of the user's slice hierarchy.
+            It usually does not contain units directly, but may be used to set defaults for the whole tree.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><filename>app.slice</filename></term>
```

Comment:
bikeshedding: do we really want to abbreviate that? why not `applications.slice`?

Reply:
It would be rather painful to change the three slice names at this point :-/

Reply:
ok

Reply:
I'd also vote for "app". It's shorter, and "app" is has already become a word of its own. ("App" is maybe not exactly the same as "application", but I don't think this distinction matters in this context.)